packages/angular-language-server/package.yaml

```diff
@@ -17,41 +17,36 @@ source:
     - typescript@5.1.3
```

Comment:
Doesn't this version need upgrading too?

Reply:
Updated

Reply:
Thanks!